android/app/src/main/assets/speakers

```diff
@@ -1,4787 +1,5254 @@
-        "mobile": null, 
-        "name": "Barry Threw", 
-        "organisation": "#NEWPALMYRA", 
-        "photo": "/images/speakers/BarryThrew_48.jpg", 
-        "position": null, 
-        "sessions": [
-            {
-                "id": 56, 
-                "title": "#NEWPALMYRA, #FREEBASSEL, and the Preservation of Cultural Heritage"
-            }
-        ], 
-        "short_biography": "", 
-        "twitter": "https://twitter.com/newpalmyraorg", 
-        "website": "http://newpalmyra.org"
-    }, 
-    {
-        "country": "Slovenia", 
-        "email": "email@domain.com", 
-        "facebook": null, 
-        "github": "", 
-        "id": 49, 
-        "linkedin": "", 
-        "long_biography": "I'm a software engineer from Maribor, Slovenia. I started working on LibreOffice as volunteer in 2012 and later joined Collabora Productivity to hack on LibreOffice as my full-time job.", 
-        "mobile": null, 
-        "name": "Tomaž Vajngerl", 
-        "organisation": "Collabora Productivity", 
-        "photo": "/images/speakers/TomažVajngerl_49.jpeg", 
-        "position": null, 
-        "sessions": [
-            {
-                "id": 58, 
-                "title": "LibreOffice 5 and beyond"
-            }
-        ], 
-        "short_biography": "", 
-        "twitter": "", 
-        "website": "http://tomazvajngerl.blogspot.si/"
-    }, 
-    {
-        "country": "Singapore", 
-        "email": "email@domain.com", 
-        "facebook": null, 
-        "github": "https://github.com/georgegoh", 
-        "id": 53, 
-        "linkedin": "https://www.linkedin.com/in/georgegoh", 
-        "long_biography": "George Goh is a technical professional who has been programming for close to 15 years, spanning domains from embedded systems, to mobile, to the cloud. Currently a senior consultant for Red Hat's cloud business in ASEAN, he has a passion for solving problems the open source way.", 
-        "mobile": null, 
-        "name": "George Goh", 
-        "organisation": "Red Hat", 
-        "photo": "/images/speakers/GeorgeGoh_53.jpg", 
-        "position": null, 
-        "sessions": [
-            {
-                "id": 62, 
-                "title": "How I left my word processor and embraced Asciidoc"
-            }
-        ], 
-        "short_biography": "", 
-        "twitter": "https://twitter.com/georgegoh", 
-        "website": "https://www.linkedin.com/in/georgegoh"
-    }, 
-    {
-        "country": "United States", 
-        "email": "email@domain.com", 
-        "facebook": null, 
-        "github": "https://github.com/tusuzu", 
-        "id": 57, 
-        "linkedin": "https://ww
```

Comment:
Isn't this URL working for speaker images?
Only a single line of change in the adapter will fix this ;)

https://github.com/fossasia/open-event-android/blob/development/android/app/src/main/java/org/fossasia/openevent/adapters/SpeakersListAdapter.java#L95